packaging/package/manifest.py

```diff
@@ -0,0 +1,144 @@
+"""
+Module for JAR manifest.
+"""
+
+import os
+
+from collections import OrderedDict
+from six import BytesIO
+
+
+class ManifestKeyException(Exception):
+
+    pass
+
+
+class ManifestSection(OrderedDict):
```

Comment:
Have you thought about giving ManifestSection an OrderedDict member variable instead of using inheritance? That would make it more flexible.